extensions/currency-exchange/src/index.tsx

```diff
@@ -384,14 +387,13 @@ function enrichExchangeData(
   return currencyData as CurrencyResult;
 }
 
-function getFlagEmoji(countryCode: string): string {
-  const codePoints = countryCode
+function getFlagEmoji(countryAndRegionCode: string): string {
+  const codePoints = countryAndRegionCode
     .toUpperCase()
     .split("")
     .map((char) => 127397 + char.charCodeAt(0));
-  let v = countryCode.startsWith("X") ? "🇺🇳" : String.fromCodePoint(...codePoints) || "🇺🇳";
-  v = countryCode === "TW" ? "🇨🇳" : v;
-  v = countryCode === "AN" ? "🇺🇳" : v;
+  let v = countryAndRegionCode.startsWith("X") ? "🇺🇳" : String.fromCodePoint(...codePoints) || "🇺🇳";
```

Comment:
Not to start another flame war, but XK (Kosovo) usually has its own emojis. It won’t be a problem for currency, but as a general code-to-emoji function it ain’t right.

Reply:
XK is not in currency support list.